### DISCUSSION TITLE
Brands - taking them a step further

### DISCUSSION THREAD

**Discussion by glennjacobs** [lunarphp/lunar#1232] [Feature Requests]
In the upcoming v0.6 release Brands now get custom attributes, just like Products and Collections.

Further considerations for brands include...

### Multiple Brands per-product
If you take for example a football (soccer) shirt. It could have more than one associated brand, e.g. Nike + Manchester United.
Currently this isn't possible in Lunar, and it probably should be.

**Comment by Navi2016** [comment]
Isn't the brand still just Nike? (Or the actual manufacturer of the shirt) I've never seen products with multiple brands.

**Reply by glennjacobs** [reply]
Brands in Lunar are more about search than inventory or manufacturing. If you like Manchester United you'd want to see the shirt, just as you would want to see if you like Nike.
